script/Deploy.s.sol

```diff
@@ -4,9 +4,82 @@
 pragma solidity 0.8.20;
 
 import {Script} from "forge-std/Script.sol";
+import {console2} from "forge-std/console2.sol";
+import {CreatorTokenFactory} from "src/CreatorTokenFactory.sol";
+import {IShowtimeVerifier} from "src/lib/IShowtimeVerifier.sol";
 
 contract Deploy is Script {
+  CreatorTokenFactory creatorTokenFactory;
+  uint256 constant EXPECTED_NONCE = 0; // TODO Edit this with the nonce of the deployer address
+    // The list of networks to deploy to.
+  string[] public networks = ["mainnet", "base"];
+  // TODO Edit this with the address of the ShowtimeVerifier contract
+  IShowtimeVerifier[] VERIFIERS = [
+    // ShowtimeVerifier contract on mainnet
+    IShowtimeVerifier(0x481273EB2B6A21e918f6952A6c53C08691FE768F),
+    // ShowtimeVerifier contract on base
+    IShowtimeVerifier(0x481273EB2B6A21e918f6952A6c53C08691FE768F)
+  ];
+  // TODO Edit this with the domain separator of the ShowtimeVerifier contract on mainnet
+  bytes32 constant MAINNET_DOMAIN_SEPARATORS = "MAINNET_DOMAIN_SEPARATORS";
+  // TODO Edit this with the domain separator of the ShowtimeVerifier contract on base
+  bytes32 constant BASE_DOMAIN_SEPARATORS = "BASE_DOMAIN_SEPARATORS";
```

Comment:
Instead of hardcoding these, you should be able to read them in the script from the actual contract. You'll need to declare your verifier as `ITestableShowtimeVerifier` instead.

Reply:
Gotcha, implemented here: [a137aee](https://github.com/showtime-xyz/creator-tokens/pull/40/commits/a137aee05fb7e11bd3357dec0446190d99c0cd88)